.github/workflows/builds.yml

```diff
@@ -357,6 +357,9 @@ jobs:
           node-version-file: ".nvmrc"
       - name: Install pnpm
         run: npm i -gf "pnpm@$(node -p 'require(\"./package.json\").engines.pnpm')" && pnpm -v
+      - name: Tweak pnpm.cjs
```

Comment:
This tweak is for 
https://github.com/electron-userland/electron-builder/issues/6933#issuecomment-1213438889

Reply:
If this is only needed while we wait for a fix from upstream (based on the above comment), should we at least have a `TODO` comment with the above link - so that we can revisit and remove later?

Reply:
Done 👍